requirements.txt

```diff
@@ -29,6 +29,7 @@ torchinfo
 filelock
 psutil
 protobuf==3.20.1 # https://github.com/databrickslabs/dbx/issues/257
+s3fs
```

Comment:
Maybe put this in a separate requirements file, i.e. `requirements_benchmarking.txt`